### PR TITLE
Created datamigration and new field for displaying saved search domain lists [PD-904]

### PR DESCRIPTION
**Pull request by almathew**
All tests pass.

### CODE REVIEW

File: seo/migrations/0009_add_default_prm_saved_search_sites.py [hunk 1]
```diff
@@ -0,0 +1,532 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
```
Comment (almathew) [inline seo/migrations/0009_add_default_prm_saved_search_sites.py]: Most of the magic happens here.

File: seo/migrations/0009_add_default_prm_saved_search_sites.py [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+        relationships_to_create = []
+
+        for company in companies:
+            for site in default_sites:
+                relationships_to_create.append((company.pk, site.pk))
+
+            job_source_ids = company.job_source_ids.all()
+            buids = job_source_ids.values_list('id', flat=True)
+            sites = orm.SeoSite.objects.filter(business_units__id__in=buids)
+            if sites.count() <= 5:
```
Comment (almathew) [inline seo/migrations/0009_add_default_prm_saved_search_sites.py]: @fezick suggested 5 as the magic number we consider them to have too many domains and force someone to do it manually. They'll have to do it manually after this point anyway, so I'm pretty ok with 5.

File: universal/helpers.py [hunk 3]
```diff
@@ -94,6 +94,7 @@ def get_company(request):
     # If settings.SITE is set we're on a microsite, so get the company
     # based on the microsite we're on instead.
     if settings.SITE.canonical_company:
+        print settings.SITE.canonical_company
```
Comment (tdphillips) [inline universal/helpers.py]: Intentional?

**Comment by almathew** [comment]
fixed

**Pull request merged**